app/main.py

```diff
@@ -1,3 +1,90 @@
+import json
+
+
 def shop_trip():
-    # write your code here
-    pass
+    my_dic = {}
+    with open('app/config.json')as file:
+        data = json.load(file)
+        my_dic.update(data)
+
+    fuel_price = my_dic["FUEL_PRICE"]
+    shops = []
+    customers = []
+    customers += (my_dic["customers"])
+    shops += (my_dic["shops"])
+    shop_list_each_name_each_store = []
+    for name in range(len(customers)):
+        sum_for_name_for_each_store = []
+        for shop in range(len(shops)):
+            price = 0
+            price += customers[name]["product_cart"]['milk'] * \
+                shops[shop]["products"]['milk']
+            price += customers[name]["product_cart"]['bread'] * \
+                shops[shop]["products"]['bread']
+            price += customers[name]["product_cart"]['butter'] * \
+                shops[shop]["products"]['butter']
+            my = customers[name]["location"]
+            mall = shops[shop]["location"]
+            dist = ((my[0] - mall[0]) ** 2 + (my[1] - mall[1]) ** 2) ** 0.5
+            car_eats = customers[name]["car"]["fuel_consumption"]
+            fuel = fuel_price * dist * car_eats * 2 / 100
+            price += fuel
+            sum_for_name_for_each_store.append(round(price, 2))
+        shop_list_each_name_each_store.append(sum_for_name_for_each_store)
+
+    for person in range(len(customers)):
+        budget = customers[person]["money"]
+        food = customers[person]['product_cart']
+        print(f"{customers[person]['name']} "
+              f"has {customers[person]['money']} dollars")
+        print(f"{customers[person]['name']}'s trip to the Outskirts Shop "
+              f"costs {shop_list_each_name_each_store[person][0]}")
+        print(f"{customers[person]['name']}'s trip to the Shop \'24/7\' "
```

Comment:
```suggestion
        print(f"{customers[person]['name']}'s trip to the Shop \'24/7\' "
```

create variable for {customers[person]['name'] and don't use \ in  \'24/7\'